ruby_on_rails/testing/unit_testing.md

```diff
@@ -0,0 +1,84 @@
+### Introduction
+
+You may remember testing from the Ruby section and the emphasis on the importance of testing. Similar to that section, testing in Ruby on Rails is important to ensure your application is stable and to catch bugs before you expand your application even more! The first kind of testing that we will cover will be unit tests.
+
+### Lesson Overview
+
+This lesson will cover a few points.
```

Comment:
```suggestion
This section contains a general overview of topics that you will learn in this lesson.
```
I believe we should stick to the wording suggested in the style guide 
https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md